src/linkgraph/linkgraphschedule.cpp

```diff
@@ -119,7 +120,7 @@ void LinkGraphSchedule::SpawnAll()
 /* static */ void LinkGraphSchedule::Clear()
 {
 	for (JobList::iterator i(instance.running.begin()); i != instance.running.end(); ++i) {
-		(*i)->JoinThread();
+		(*i)->AbortJob();
```

Comment:
I don't think you can loose the `JoinThread` here. It is required to call either `join` or `detach` on a `std::thread` before destruction.

Reply:
JoinThread is called in ~LinkGraphJob, it's not necessary to call it twice.

Reply:
Ah, sorry, somehow missed that.